tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AspNetCore/UserEvents/SignInManagerPasswordSignInIntegration.cs

```diff
@@ -82,17 +84,15 @@ internal static TReturn OnAsyncMethodEnd<TTarget, TReturn>(TTarget instance, TRe
 
             if (security.IsAnonUserTrackingMode)
             {
-                var anonId = UserEventsCommon.GetAnonId(id);
-                if (anonId is not null)
-                {
-                    tryAddTag(Tags.AppSec.EventsUsers.LoginEvent.FailureUserId, anonId);
-                }
-
+                var loginAnon = UserEventsCommon.Anonymize(login);
+                tryAddTag(Tags.AppSec.EventsUsers.LoginEvent.FailureUserLogin, loginAnon!);
```

Comment:
is the ! in loginAnon! required? method Anonymize returns a non nullable string

Reply:
yes, I m not sure why the compiler doesn't like it with the func `tryAddTag`

Reply:
I _suspect_ you can remove this now? 🤔 `GetAnonId` used to return a `string?`, but `Anonymize` returns ` string`.

Reply:
yes removed in 03da6074d4a68f6088e3c7119b6fb5c29b345c1d